modules/c++/io/tests/serializeTest1.cpp

```diff
@@ -73,7 +73,9 @@ class A : public Serializable
     {
 	std::string toFill;
 	sys::byte b[1];
-	while (true)
+	// Work-around warning C4127: conditional expression is constant;
+	// see https://msdn.microsoft.com/en-us/library/6t66728h.aspx and http://stackoverflow.com/questions/1946445/c-c-how-to-use-the-do-while0-construct-without-compiler-warnings-like-c412
+	for (;;) // while (true)
```

Comment:
/wd4127 can find other less obvious situations than "while (true)".  The warning itself can be somewhat interesting because in a situation like "if (... something that is true at compile time ...)" the "else" (if any) will never be executed, which might indicate dead code or a programming error.

Yes, I would also prefer "while (true)" which is why I left it in as a comment.